lib/payment/response_classes/merchant_stripe.rb

```diff
@@ -188,7 +188,8 @@ def <=>(other)
     end
 
     attr_accessor :charge_id, :status, :amount_cents, :currency,
-                  :customer_name, :credit_card_brand, :created_at
+                  :customer_name, :customer_email, :credit_card_brand,
```

Comment:
I think I have to add here all the detailed response attributes too, if we want to use this class to represent:

* GET charge response (not needed in core now)
* GET dispute response's charge part (we need only the customer_email now)